### PR TITLE
Update ttml-imsc1-errata.html

### PR DESCRIPTION
**Pull request by tmichel07**
Added a first errata for missing Ack section

### CODE REVIEW

File: spec/ttml-imsc1-errata.html [hunk 1]
```diff
@@ -1,24 +1,119 @@
...
+    <p>The IMSC1 Recommendation is mission an Acknowledgments section.</p>
+    <p class="description"><em><strong><span class="title-correction">Resolution</span></strong></em>:</p>
+    <p>Add the following section:</p>
+    <div class="diff-add" style="margin:5px; border: 1px solid red; padding:10px;">
+      <h2>Acknowledgments (Non-Normative)</h2>
+      <p>The editor acknowledges the current and former members of the Timed
+        Text Working Group, the members of other W3C Working Groups, and
+        industry experts in other forums who have contributed directly or
+        indirectly to the process or content of this document as follows:</p>
+      <p>Glenn Adams, Skynav Inc.; John Birch, Invited expert; Cyril Concolato,
```
Comment (nigelmegitt) [inline spec/ttml-imsc1-errata.html]: I'm not at all sure that all of these people would want or expect their names on here. Is it normal to check with people first? It's very clear to me that several names here had zero involvement with IMSC at any stage.

Comment (tmichel07) [inline spec/ttml-imsc1-errata.html]: Nigel,

I have copied the participants listed in the current TTWG.

We could also say that we thanks the participants of the TTWG (without 
any names) and add a list of contriobutor names like:
WDYT ?

--------------------------------------

The editor acknowledges the current and former members of the Timed Text 
Working Group, the members of other W3C Working Groups, and industry 
experts in other forums who have contributed directly or indirectly to 
the process or content of this document.


The editor wishes to especially acknowledge the following contributions 
by members: Glenn Adams, Skynav Inc.; Mike Dolan, Invited expert; Nigel 
Megitt, British Broadcasting Corporation; Thierry Michel, W3C; Andreas 
Tai, IRT. (more names here ???)

The editor also wishes to  acknowledge Digital Entertainment Content 
Ecosystem (DECE).

------------------------------

Comment (palemieux) [inline spec/ttml-imsc1-errata.html]: > We could also say that we thanks the participants of the TTWG (without any names)

Yes.

File: spec/ttml-imsc1-errata.html [hunk 2]
```diff
@@ -1,24 +1,119 @@
...
+    <h2>Rejected proposed erratas:</h2>
+    <ul>
+      <li>R01: <span style="color: #034575;">None</span></li>
+    </ul>
+    <hr>
+    <h2>Erratas:</h2>
+    <p id="E01" class="error"><strong>- E01: </strong> Correction in the IMSC1
+      Annex section (published 22 04 2016)</p>
+    <p class="description"><span class="title-description">Description</span>:</p>
+    <p>The IMSC1 Recommendation is mission an Acknowledgments section.</p>
```
Comment (palemieux) [inline spec/ttml-imsc1-errata.html]: IMSC1 does not include an Acknowledgements section.

File: spec/ttml-imsc1-errata.html (hunk 1 above)
Comment (palemieux) [inline spec/ttml-imsc1-errata.html]: > add a list of contriobutor names like: WDYT ?

I do not understand what this means.

File: spec/ttml-imsc1-errata.html (hunk 2 above)
Comment (nigelmegitt) [inline spec/ttml-imsc1-errata.html]: oh I see: s/mission/missing

**Pull request merged**